### ISSUE TITLE
Suggestion Line

### ISSUE THREAD

**Issue by Em4nu3lV1ct0r** [Vicen04/Dw1DataAndPatches#9]
Thank you for the excellent work on this beloved game, I made this line of suggestions, I hope you like some of them or that this gives you new ideas for future updates.

Suggestions for changes based on Digimon lore, Digimon Reference Book and crazy ideas I had while finishing the Game with mod.

------------------------------------------------Suggestions for Changes-----------------------------------------------

----Digimon Chart----

*Digimons Fresh and Training in same line to open a 3rd line of ultimates (I don't know if it's possible but it would be really cool haha);
![Digimon Chart Mod Idea](https://github.com/user-attachments/assets/49446e55-dd3d-48e2-8a1e-4288349f89a0)

----Digimon's----

*Gabumon: Restaure Original Line Digivolute for Ogremon;

*Meramon: Restaure Original Line Digivolute for Andromon;

*Mamemon: Remove Digi-Evolution Line from Leomon;

*SkullGreymon:  Remove Digi-Evolution Line from Garurumon;

*MetalMamemon:  Remove Digi-Evolution Line from Frigimon;

*Panjyamon: Update for Ultimate (he is Ultimate in Digimon Reference Book), Digivolve from: Leomon (Mamemon Slot), Garurumon (SkullGreymon Slot) and Frigimon (MetalMamemonSlot), using status of Weregarurumon;

*Gigadramon: Digivolve from: Drimogemon (Extra Line), Coelamon (Extra Line), and continue with Birdramon(Extra Line);

*MetalEtemon: Only available in Factorial Town Guardromon Update (MetalEtemon having multiple Digi-evolution lines and Etemon only having one is weird, making him exclusive to the Factorial Town Guardromon Update is cooler);

*Machinedramon: Only available in Factorial Town Guardromon Update;

---- Factorial Town Guardromon Update----

*Remove: Elecmon, Leomon and Greymon;

*No Changes: Devimon;

*Seadramon: Factorial Town Guardromon update for Gigadramon (50%) or Megaseadramon (10%);

*Nanimon: Factorial Town Guardromon update for Giromon (30%) or Piximon (30%);

*Mamemon: Factorial Town Guardromon update (100%) for MetalMamemon;

*MetalEtemon: Factorial Town Guardromon update exclusive (100%) from Etemon;

*Machinedramon: Factorial Town Guardromon update exclusive (20%) from Metalgreymon, Megadramon, Gigadramon, Andromon and Metalmamemon (In Digimon lore, Machinedramon was made from pieces of these Digimon's) only available after finishing the game(I don't know if it's possible).

----Skill----

*Metalmamemon: Remove War Cry, add DG Dimension with Reverse Prog movement.

----Inhabitant Function----

*Jijimon: delivers the Machi Coder (in place of the Chain Melon) after Machinedramon's defeat in Back Dimension.

----Drop Rate----

*Increase chance of finding rare fruits (except those that can be obtained in Digimon Curling or Digimon's drop);

*Increase chance for Digi-evolution Items in Drimogemon Treasure;

----Bonus Area----

*Map color fix already integrated;

*Bosses explain that the place is a digital distortion created by Analogman;

*Box in bônus área: Chain Melon;

*Items on the floor in the bonus area: all Gold Acorn.

----New Bosses----

*H-Kabuterimon: it will teach you "Bug" regardless of your digimon;

*Andromon: it will teach you "DG Dimensional" regardless of your digimon.

----Arena Lobby----

*Limited Real Cup for Champions and Ultimates (It's weird to battle baby Digimon);

*Real Cup Rewards: Giga Hand (50%) and Noble Mane (50%) amost like a S Cup;

*Merit Points Rewards for Winning Cups: R(100), S(80) ,A(40) , B,C,D,(10) Other Copa(60).

----Digimon Curling----

*Increase chance for Digi-evolution Items;

*Penguinmon Rewards: Silver Ball, Coral Charm, Mysty Egg, Ice Mushrm and Power Ice;

*Metalmamemon Rewards: Metal Armor, Chainsaw, Small Spear, Power fruit, Muscle Yam, Speed Leaf, Sage fruit and Calm Berry;

*Merit Points Rewards for Winning the Curling: 5 or 10 points.

----Shogun Gekomon Merit Points Store----

*Lower the price of Chips from 800 to 300 or 250 Merit Points;
*Lower the price of Digi-evolution items from 500 to 400 or 350 Merit Points.

---Extra patches ---

*Sell Spiny Green (very useful for training) instead of Pricklypear (Supercarrot already do something similar) in Useful - Helpful Items;

*Remove tamer level in Useful - Stat gains evo item;

*Myotismon patch substitute other digimons (Panjyamon or Vademon Maybe), getting Machinedramon and Myotismon in the same save game would be cool;

-------------------------------------------------------------------------------------------------------------------------------------

Not much haha, I changed it several times, but I think it's the end, I hope it's useful.

I'm sorry for any language mistakes, English is not my native language.

**Comment by Vicen04** [comment]
* "Tech boost doesn't seem to be working as it should".  I need an explanation for this one, have you tried to use the same technique against the same digimon in vanilla and then in the hack to see the difference? In which way is not working as intended?
* "Machinedramon is dropping Machi Coder, but there is no way to get it because we are immediately teleported to File City after the battle"  That is not a bug.
* "The bonus area fix doesn't seem to work when reaching 100 prosperity" I just tried it with 100 prosperity and it triggered. This fix can only happen once per savefile, it will not trigger again or that is how it is supposed to work, it was made for people that are using a savefile from a different modded version, and old Vice version or vanilla, you can technically cheat to trigger it again...
* "The items and Bosses in the bonus area reset if you manage to keep going back and forth in it, but I think they should only reset the next time Machinedramon appears" That is just how maps works, that is the reason (and I even state that in the video), why the bonus area is made to be only accessible once per Machinedramon appearance. Losing a live to enter again is up to the player.
* "Sometimes the Gold acorn in the middle of the bonus area (near the sign) becomes uncollectible because we can't get close enough to it"  Yes, I'm fully aware that one is difficult to get, it will stay that way.

**Comment by Vicen04** [comment]
Hi, 
Seems that the error only happens with Duckstation (or at least only with that from the multiple emulators I have), I just made an update that should fix the issue.
Also, the tech boost works was only not working for your partner Digimon, if you were playing either Hardmode or Hardcore, the boost would work in NPC.

Thanks for letting me know.

As for the other ideas, some are not exactly possible, some others would just be too much work. I will think about what to do when I decide to keep going with the hack.

**Comment by Em4nu3lV1ct0r** [comment]
I understand, I'm happy to contribute, I even found another small error, I don't know if you've noticed:
![Picsart_24-08-26_20-41-58-472](https://github.com/user-attachments/assets/e6898890-398b-4ff3-8d2e-47c186d71357)

*Ogremon still appears as Gabumon's Digi-evolution, but Pangyamon has taken that slot;

*Andromon still appears as Meramon's Digi-evolution, but MetalEtemon has taken that slot.

**Comment by Vicen04** [comment]
I'll fix that one on my next big update, I forgot to just remove them from the preevolution chart, it does not cause any kind of issue. Something similar happens in the Filth patch, were the old digivolutions still show up.

**Comment by Em4nu3lV1ct0r** [comment]
bro, I continued testing the last update, the enemies are hitkill lol

![capture giga](https://github.com/user-attachments/assets/fe0ad324-e239-4614-9031-b913f43528d9)

I don't know if I did something wrong, but it's good to check.


I'm sorry for making this headache for you ;(

**Comment by Vicen04** [comment]
I just want to make sure, did you apply the patches to a clean USA version or it was over the old Vice version?

**Comment by Em4nu3lV1ct0r** [comment]
Clean Version

**Comment by Vicen04** [comment]
Okay, there was 1 byte with a value off, it should have been 16 rather than 15.
![image](https://github.com/user-attachments/assets/2032e568-427e-4f9a-86ba-ca30d9d0f304)

(This was a 0F before, it is supposed to be a 10)
It was literally that.
It also only affects the base game, all the other difficult versions were working fine.

It has been updated again and I tested it again, it works fine, Vademon also does at least twice the damage with Metal Sprinter with the save you provided.

Well, thanks for that again.
